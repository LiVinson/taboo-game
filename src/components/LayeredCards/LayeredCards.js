import { LayeredTabooCard } from 'components/shared/TabooCard'
import React from 'react'

const LayeredCards = () => {
	return (
		<React.Fragment>
			<LayeredTabooCard layer="top" tabooWord="Taboo!" list={['Taboo!', 'Taboo!', 'Taboo!', 'Taboo!']} />
			<LayeredTabooCard layer="middle" tabooWord="Taboo!" list={['Taboo!', 'Taboo!', 'Taboo!', 'Taboo!']} />
			<LayeredTabooCard layer="bottom" tabooWord="Taboo!" list={['Taboo!', 'Taboo!', 'Taboo!', 'Taboo!']} />
		</React.Fragment>
	)
}

export default LayeredCards
