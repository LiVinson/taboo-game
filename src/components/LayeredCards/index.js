import LayeredCards from "./LayeredCards"
export default LayeredCards;