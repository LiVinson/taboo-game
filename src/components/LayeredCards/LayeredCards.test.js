import React from "react"
import renderer from "react-test-renderer"
import LayeredCards from "./LayeredCards"


it("renders correctly", ()=> {
    const LayeredCardsComponent = renderer.create(<LayeredCards />).toJSON()
    expect(LayeredCardsComponent).toMatchSnapshot()
})