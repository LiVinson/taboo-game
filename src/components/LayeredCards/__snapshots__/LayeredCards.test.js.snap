// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <div
    className="taboo-card taboo-card--middle"
  >
    <div
      className="taboo-card__word-container"
    >
      <p
        className="taboo-card__word"
      >
        Taboo!
      </p>
    </div>
    <div
      className="taboo-card__list-container"
    >
      <ul
        className="taboo-card__list"
      >
        <li
          className="taboo-card__list-item"
        >
          Taboo!
        </li>
        <li
          className="taboo-card__list-item"
        >
          Taboo!
        </li>
        <li
          className="taboo-card__list-item"
        >
          Taboo!
        </li>
        <li
          className="taboo-card__list-item"
        >
          Taboo!
        </li>
      </ul>
    </div>
  </div>,
  <div
    className="taboo-card taboo-card--bottom"
  >
    <div
      className="taboo-card__word-container"
    >
      <p
        className="taboo-card__word"
      >
        Taboo!
      </p>
    </div>
    <div
      className="taboo-card__list-container"
    >
      <ul
        className="taboo-card__list"
      >
        <li
          className="taboo-card__list-item"
        >
          Taboo!
        </li>
        <li
          className="taboo-card__list-item"
        >
          Taboo!
        </li>
        <li
          className="taboo-card__list-item"
        >
          Taboo!
        </li>
        <li
          className="taboo-card__list-item"
        >
          Taboo!
        </li>
      </ul>
    </div>
  </div>,
]
`;
